app/models/pipeline_run.rb

```diff
@@ -550,18 +550,21 @@ def db_load_amr_counts
     unless File.size?(amr_results) < 10
       amr_results_table = CSV.read(amr_results, headers: true)
       amr_results_table.each do |row|
-        amr_counts_array << {
+        amr_count_for_sequence = {
           gene: row["gene"],
           allele: row["allele"],
           coverage: row["coverage"],
           depth: row["depth"],
-          annotation_gene: row["annotation"].split(";")[2],
-          genbank_accession: row["annotation"].split(";")[4],
           drug_family: row["gene_family"],
           total_reads: row["total_reads"],
           rpm: row["rpm"],
           dpm: row["dpm"],
         }
+        if row["annotation"]
```

Comment:
I think you might want to still check if `row["annotation"].present?` unless you're sure that CSV.read wouldn't generate `{"annotation": ""}`

Reply:
In that case, should we also check if split has at least len 5? What can we assume about the output?
(As a side note/reference, when we expect a certain format we are better off using a regex to map and extract the content we need. We would just need to look at the regex comparison result to immediately know if the output was expected)

Reply:
I think it might almost be easier to write a python script to clean up the arg-annot fasta file.

Speaking of, srst2 updated their arg-annot file to r3 10 months ago. The chanzuckerberg fork we use is still on r2. I wonder what other features or fixes we might be missing out on :(

Reply:
In any case, I wrote the regex. Now either we get nil or something we can work with. For sure